doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -65,23 +65,43 @@ There are two implementations currently supported:
   .. figure:: images/bootloader_memory_layout.svg
      :alt: Memory layout
 
-For detailed information about the memory layout, see the partition configuration in the :file:`partitions.yml` file, located in the build folder directory, or run ``ninja partition_manager_report``.
+By default, building an application with any bootloader configuration creates a :ref:`multi-image build <ug_multi_image>`, where the :ref:`partition_manager` manages its memory partitions.
+Bootloaders are built as child images.
+When building an application with :ref:`Cortex-M Security Extensions (CMSE) enabled <app_boards_spe_nspe_cpuapp_ns>`, then :ref:`Trusted Firmware-M (TF-M) <ug_tfm>` is also built with the image automatically.
+From the bootloader perspective, the TF-M is part of the booted application image.
+
+.. _ug_bootloader_flash_static_requirement:
+
+Static partition requirement for DFU
+------------------------------------
+
+By default, the Partition Manager generates the partition map dynamically.
+As long as you are not using Device Firmware Updates (DFU), you can use the dynamic generation of memory partitions.
+
+However, if you want to perform DFU, you must :ref:`define a static partition map <ug_pm_static>`.
+This is important because the dynamically generated partitions change between builds.
+The newly generated application images may no longer be compatible with the bootloader programmed to the device.
+In such case, the newly built application image may not be bootable by the bootloader programmed onto the device.
 
 .. note::
+   For detailed information about the memory layout used for the build, see the partition configuration in the :file:`partitions.yml` file, located in the build folder directory, or run ``ninja partition_manager_report``.
    You must enable the Partition Manager to make the :file:`partitions.yml` file and the ``partition_manager_report`` target available.
-   See :ref:`partition_manager` and :ref:`ug_multi_image` for details.
 
-By default, building an application with any bootloader configuration creates a :ref:`multi-image build <ug_multi_image>`, where the :ref:`partition_manager` manages its memory partitions.
-When building an application with :ref:`Cortex-M Security Extensions (CMSE) enabled <app_boards_spe_nspe_cpuapp_ns>`, then :ref:`Trusted Firmware-M (TF-M) <ug_tfm>` is also built with the image automatically.
+   The :file:`partitions.yml` file is present also if the Partition Manager generates the partition map dynamically.
+   You can use this file as a base for your static partition map.
+
+The memory partitions that must be defined in the static partition map depend on the selected bootloader chain.
+For details, see :ref:`ug_bootloader_flash`.
 
 .. _immutable_bootloader:
 
-First-stage immutable bootloader
-================================
+Immutable bootloader
```

Comment:
This now looks like a sub-header of "Static partition requirement for DFU".
And I beleive that it is its own section right?

Reply:
It is not a sub-header of Static partition requirement, because it has higher-level header formatting === vs lower-level --- of that section.

Reply:
But it looks smaller:
![bilde](https://github.com/nrfconnect/sdk-nrf/assets/89006246/ada9fe78-5e50-404d-86d1-584535e6f162)

Reply:
There was an issue with the Architecture heading, actually. Fixed.